cypress/e2e/e2e.cy.js

```diff
@@ -0,0 +1,140 @@
+        cy.get('#messenger-name').focus().should('have.focus')
+
+        cy.get('h2').should('contain', 'How can we help you?')
+        cy.get('#messenger-name').should('have.attr', 'required');
+        cy.get('#email').should('have.attr', 'required');
+        cy.get('#message').should('have.attr', 'required');
+    })
+
+    it('Mostra e oculta uma mensagem de sucesso ao enviar o formulário do messenger com sucesso', () => {
+
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').should('be.visible')
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').click()
+        cy.get('div[class="Messenger_box__qUF2H"]').should('be.visible')
+        cy.get('div[class="Messenger_box__qUF2H"]').click()
+        cy.get('h2').should('contain', 'How can we help you?')
+        cy.get('input[id="messenger-name"]').type('Seu Madruga')
+        cy.get('#email').type('madruguinha@gmail.com')
+        cy.get('#message').type('chiforinfola')
+        cy.get('button[type="submit"]').click()
+        cy.get('div[role="alert"].Messenger_success__GOoJk')
+            .should('be.visible')
+            .and('contain', 'Your message has been sent.')
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').click()
+        cy.wait(3000)
+        cy.get('div[role="alert"].Messenger_success__GOoJk').should('not.exist')
+    })
+
+    it('Limpa todos os campos do formulário do messenger ao preenchê-los, fechar o messenger e abri-lo novamente', () => {
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').should('be.visible')
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').click()
+        cy.get('div[class="Messenger_box__qUF2H"]').should('be.visible')
+        cy.get('div[class="Messenger_box__qUF2H"]').click()
+        cy.get('h2').should('contain', 'How can we help you?')
+        cy.get('input[id="messenger-name"]').type('Seu Madruga')
+        cy.get('#email').type('madruguinha@gmail.com')
+        cy.get('#message').type('chiforinfola')
+        cy.get('input[id="messenger-name"]').clear()
+        cy.get('#email').clear()
+        cy.get('#message').clear()
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').click()
+        cy.get('button[class="Messenger_openCloseButton__6O7-y"]').click()
+        //validando se os campos foram limpos
+        cy.get('input[id="messenger-name"]').should('have.value', '')
+        cy.get('#email').should('have.value', '')
+        cy.get('#message').should('have.value', '')
+    })
+
+})
+describe('Testes em Viewport Móvel', () => {
+    beforeEach(() => {
+
+        cy.viewport('iphone-6')
+        cy.visit('http://localhost:3000')
+        cy.contains('button', 'Accept').click()
+    });
+
+    it('Mostra as colunas Nome da Empresa e Ação, e oculta as colunas ID, Indústria, Número de Funcionários e Tamanho em um viewport móvel', () => {
+
+        cy.get('th').contains('Company name').should('be.visible')
```

Comment:
```suggestion
        cy.contains('th', 'Company name').should('be.visible')
```

O mesmo vale para as próxima linhas.